cms/static/sass/elements/_forms.scss

```diff
@@ -456,7 +456,7 @@ input.search {
   @include box-sizing(border-box);
   border: 1px solid $darkGrey;
   border-radius: 20px;
-  background: url(../images/search-icon.png) no-repeat 8px 7px #edf1f5;
+  background: url($static-path + '/images/search-icon.png') no-repeat 8px 7px #edf1f5;
```

Comment:
Please see [my comments about the formatting for passing this path variable](https://github.com/edx/edx-platform/pull/8271/files#r37541733) into rules like this.